test/SILGen/sil_locations.swift

```diff
@@ -109,7 +109,7 @@ func multipleReturnsImplicitAndExplicit() {
     return
   }
   x += 1
-  // CHECK-LABEL: sil hidden [ossa] @$s13sil_locations34multipleReturnsImplicitAndExplicityyF
+  // CHECK-LABEL: sil hidden [ossa] @$s13sil_locations34multipleReturnsImplicitAndExplicitlyyF
```

Comment:
This is not a typo, note that `yyF` suffix is part of the mangle name of `multipleReturnsImplicitAndExplicit` the same way `@$s13sil_locations14testMethodCallyyF` is for `testMethodCall`